tests/myrapname.test.js

```diff
@@ -1,6 +1,84 @@
+const { chromium } = require('playwright')
+
 describe("Myrapname.com", () => {
+  let browser
+  let page
+
+  beforeAll(async () => {
+    browser = await chromium.launch({headless: false, slowMo: 50})
+    page = await browser.newPage()
+
+    if(!page) throw new Error("No Connection Established")
+
+    await page.goto("https://www.myrapname.com", {
+      waitUntil: "networkidle0"
+    });
+  });
+
+  afterAll(async () => {
+    await browser.close();
+  });
+
+  it("should load myrapname.com", async () => {
+    expect(page).not.toBeNull();
+    expect(await page.title()).toBe('My Rap Name - A generator to automatically make rapper names')
+  });
+
+  it("should display a form", async () => {
+    expect(await page.click('form'))
```

Comment:
Is this actually testing the visibility? Maybe replace some of these expects using `.click` to `.isVisible`